oaknational/Oak-Web-Application
# fix: 975 modal navigation buttons alignment

## Description

Fixed modal navigation buttons moving vertically at tablet width when navigating between leaders on About Leadership page. I set a minimum height to the leaders' descriptions to avoid the buttons moving.

## Issue(s)

Fixes #975

## How to test

1. Go to {cloud link}/about-us/leadership
2. Open the bio modals
3. Make your screen width between 1279px and 750px.
4. Navigate through the modals using the forward and back button
5. The buttons (left and right arrows) should stay fixed 

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
